src/main/java/de/tum/in/www1/artemis/domain/modeling/ModelCluster.java

```diff
@@ -0,0 +1,76 @@
+package de.tum.in.www1.artemis.domain.modeling;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+
+/**
+ * A TextCluster.
```

Comment:
Was this intended for  "A ModelCluster" maybe ?
```suggestion
 * A ModelCluster.
```